data/remnant/remnant 1 introduction.txt

```diff
@@ -3742,3 +3743,63 @@ mission "Remnant: Expanded Horizons Storms 1"
 			label stormend
 			`	"Thanks for the ride, Captain. I have to get working on this data, but if you are interested in helping some more, I hope to have more research expeditions planned soon."`
 
+
+
+mission "Remnant: Logistics 1"
+	name "Rush delivery to <planet>"
+	description `Deliver <cargo> to <destination> by <date>.`
+	source "Caelian"
+	destination "Viminal"
+	cargo "domestic reactors" 30
+	deadline 2
+	to offer
+		has "remnant: blood test pure"
+		has "Remnant: Defense 1: done"
+	on offer
+		conversation
+			`You step out of the <ship> and look again at the strange sight of the <origin> spaceport. You still aren't used to a structure that looks as much like it grew from the landing pads as it was built next to them. The dock side entrance is busy; when your commlink vibrates, you have to find a place where the traffic eddies and you can step aside and learn that you need to check in with the information desk.`			
+			`	The woman working the desk seems startled to see you when you walk up. You can see the realization dawn on her as she scans your clothes. "Ah," she sings, "you are the ancestral sojourner among us."`
+			`	You are getting used to being gawked at, so you take it in good humor and respond by singing, "The Embers guided me to friends I didn't know, and your message invited me to meet a new one."`
```

Comment:
```suggestion
			`You step out of the <ship> amidst the strange sight of the <origin> spaceport. The structures that surround the landing pads looks as much grown as built, and they see to merge into the landing pads themselves. As usual, the dock entrance is busy; when your commlink vibrates, you have to find a place where the traffic eddies and you can step aside and learn that you need to check in with the information desk.`			
			`	The woman working the desk seems startled to see you when you walk up. You can see the realization dawn on her as she looks you over with a practised eye. "Ah," she sings, "you are the ancestral sojourner among us."`
			`	It is typical for people to gawk at, so you respond by singing, "The Embers guided me to friends I didn't know, and your message invited me to meet a new one."`
```

@W1zrad Could you take a look at this adjusted text please?

Reply:
```suggestion
			`You step out of the <ship> amidst the strange sight of the <origin> spaceport. The structures that surround the landing pads look as much grown as built, and they seem to merge into the landing pads themselves. As usual, the dock entrance is busy; when your commlink vibrates, you have to find a place where the traffic eddies so you can step aside. When you do, you learn you need to check in with the information desk.`
			`	The woman working the desk seems startled to see you when you walk up. You can see the realization dawn on her as she looks you over with a practiced eye. "Ah," she sings, "you are the ancestral sojourner among us."`
			`	It is typical for people to gawk at you, so you respond by singing, "The Embers guided me to friends I didn't know, and your message invited me to meet a new one."`
```
Just a few edits for flow and proofreading. Looks good!